src/src/lms/lms.controller.ts

```diff
@@ -65,4 +68,19 @@ export class LMSController {
 	) {
 		return this.lmsService.searchTestTracking(searchLMSDto, response);
 	}
+
+	//create certificate detail
+	@Post('/certificate/download')
+	@UseGuards(new AuthGuard())
+	public async issueCertificate(
```

Comment:
is it issueCertificate or downloadCertificate?

Reply:
changed it to downloadCertificate